docs/docs/features/combos.md

```diff
@@ -25,7 +25,7 @@ Combos configured in your `.keymap` file, but are separate from the `keymap` nod
 
 - The name of the combo doesn't really matter, but convention is to start the node name with `combo_`.
 - The `compatible` property should always be `"zmk,combos"` for combos.
-- `timeout-ms` is the length of the window (in milliseconds) in which all keys of the combo must be pressed in order to successfully trigger the combo.
```

Comment:
I think this is undoing the changes from #1277.

Reply:
I reworded this to say the same thing in fewer words so I could use the same wording in both this page and config/combos.md without the table getting really long.